components/standing/PlayerinfoBox.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "./Playerinfo.module.css";
+import UserPicture from "./PlayerinfoPicture";
+import UserPick from "./PlayerinfoPick";
+
+export type PlayerinfoProps = {
+  picture: string;
+  username: string;
+  pick: string;
+  pickLogo: string;
+  opponent: string;
+  opponentLogo: string;
+};
+
+export default function PlayerinfoBox({
+  picture,
+  username,
+  pick,
+  pickLogo,
+  opponent,
+  opponentLogo,
+}: PlayerinfoProps) {
+  return (
+    <section className={`${styles.playerinfoBox}`}>
+      <UserPicture picture={picture} />
+      <p className={`${styles.playerinfoUsername}`}>{username}</p>
+      <UserPick
```

Comment:
This style naming is not how it is supposed to be with `module.css`-files. 
Take a look at the `Use CSS Modules in React` Paragraph: 
https://www.triplet.fi/blog/practical-guide-to-react-and-css-modules/